### DISCUSSION TITLE
"图像格式转换"功能无法转换svg文件

### DISCUSSION THREAD

**Discussion by lvyangdev** [he3-app/he3-feedback#215] [Bug Report]
版本: v1.3.17 

更新了两个版本不修复，之前提过：[图片格式转换器中从svg格式转换到其他格式失败](https://github.com/he3-app/he3-feedback/issues/141)

![screenshots](https://github.com/he3-app/he3-feedback/assets/25558125/39211863-772a-46b2-a2c4-8974339a027a)

**Comment by lyzhang1999** [comment]
感谢反馈：https://github.com/he3-app/he3-feedback/issues/219

**Comment by congcongnalian** [comment]
非常高兴您能提出反馈！该问题已经修复。点击最新链接查看 图像格式转换： https://t.he3app.com?ag1i
